code/game/objects/items/weapons/implants/implant_health.dm

```diff
@@ -12,3 +12,12 @@
 	else
 		healthstring = "[round(imp_in.getOxyLoss())] - [round(imp_in.getFireLoss())] - [round(imp_in.getToxLoss())] - [round(imp_in.getBruteLoss())]"
 	return healthstring
+
+/obj/item/implanter/health
+	name = "implanter (health)"
+	implant_type = /obj/item/implant/health
+
+/obj/item/implantcase/health
+	name = "implant case - 'Health'"
+	desc = "A glass case containing a health implant."
+	implant_type = /obj/item/implant/health
```

Comment:
Going to be honest, I'm not really sure what this "health" implant does, but I thought it was weird that it lacked any implanter/cases despite seemingly being a valid admin spawn implant. If this is deemed unwanted I can remove these health implanter/case